clicommand/bootstrap.go

```diff
@@ -362,6 +363,11 @@ var BootstrapCommand = cli.Command{
 			Usage:  "Disables the Job API, which gives commands in jobs some abilities to introspect and mutate the state of the job.",
 			EnvVar: "BUILDKITE_AGENT_NO_JOB_API",
 		},
+		cli.StringSliceFlag{
+			Name:   "disable-warnings-for",
+			Usage:  "A list of warning IDs to disable",
+			EnvVar: "BUILDKITE_AGENT_NO_WARN_FOR",
```

Comment:
Should the env var be renamed too? It's certainly longer but hat seems less confusing as a user if the CLI arg and env var match
```suggestion
			EnvVar: "BUILDKITE_AGENT_DISABLE_WARNINGS_FOR",
```

Reply:
yes def! good point, forgot that in my find-and-replace spree 😅